garage/misc/tensorboard_output.py

```diff
@@ -67,9 +65,9 @@ def dump_tensorboard(self, step=None):
         self._dump_histogram(run_step)
         self._dump_tensors()
 
-    def record_histogram(self, key, val):
+    def record_histogram(self, key, val, name=None):
         if str(key) not in self._histogram_ds:
-            with enclosing_scope(self._name, "record_hist"):
+            with tf.name_scope(name, "record_hist"):
                 self._histogram_ds[str(key)] = tf.Variable(val)
             self._histogram_summary_op.append(
```

Comment:
shouldn't adding to the summary op also go inside the scope?

Reply:
Yes, I moved the scope one line up to cover both the variables and summary.merge